week1/LESSONPLAN.md

```diff
@@ -0,0 +1,66 @@
+# Lesson Plan Week 1
+
+## Agenda
+
+The purpose of this class is to introduce the student to the most basic React concepts:
+
+- How to look at a webpage from the React point of view
+- How to write reusable components and why
+- The purpose of JSX
+- What are props and how to pass them down components
+- Most important: React is just a simple library that allows you to make dynamic, modern user interfaces
+
+In order to keep it as simple as possible, please use the `React CDN` instead of create-react-app. For more info check the following [link](https://reactjs.org/docs/add-react-to-a-website.html#add-react-in-one-minute).
+
+_Explain to students what each library does: React, ReactDOM and Babel. Then ask the to download the same files and follow along._
```

Comment:
iirc, we decided to use create-react-app and later in 4th week with extra time explain the cra under the hood

Reply:
True. The lesson plans are the only thing not updated yet.